### PR TITLE
Capstone

### PR DESCRIPTION
**Pull request by AlejandroNeira98**
### In this capstone project Pull request I took into account:

- Choosing a topic that is different than the one in the original design.
- Following these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:
  - Colors.
  - Typography: font face, size, and weight.
  - Layout: composition and space between elements.
 
- The pages look almost identical to the original design. Small adjustments like text and image changes are included.

- I build these 2 pages:
  - The *home page*.
  - The *about page*.
  
- Each of these pages have versions for 2 different screen sizes: 
  - Mobile: up to 768px wide.
  - Desktop: 768px or wider.

- Interactions
  - Links
    - The *home page* has a link in the menu to the *about page*.
    - The logo in the header links to the *home page*.
    
  - Mobile menu
    - When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
 
- Dynamic page
  - The section "Featured speakers" is created dynamically in JavaScript.
  - I used a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.
  
### Here is the video presentation:

https://user-images.githubusercontent.com/93448210/152411737-640437b7-78f3-49a6-b3b4-12ec42680b20.mp4

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+      <img src="./images/partners/mozilla.png" alt="Mozilla">
+      <img src="./images/partners/Google.png" alt="Google">
+      <img src="./images/partners/navar.png" alt="Naver">
+      <img src="./images/partners/daumkakao.svg" alt="Daumkakao">
+      <img src="./images/partners/airbnb.png" alt="airbnb">
+    </div>
+  </div>
+
+  <script src="./mobile-menu.js"></script>
+  <script src="./speakers.js"></script>
```
Comment (ShoiraTa) [inline index.html]: - [ ] I have noticed that you did not include `footer` section in your project, please consider adding the light footer as provided in original design which comes after `partners` section in both pages:
image
- [ ] Please check the partners section in mobile devices screen, it does not show in `home` page: 

https://user-images.githubusercontent.com/77038610/152369083-d841abdc-309f-4d9e-b48c-4be773dde4b8.mp4

File: about.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+  <div id="mobile-menu">
+    <div id="close-btn">X</div>
+    <a href="./about.html">About</a>
+    <a href="#main-program">Program</a>
+    <a href="#">Join</a>
+    <a href="#">Sponsor</a>
+    <a href="#">News</a>
+    <a href="#">CB Campaing</a>
+  </div>
```
Comment (ShoiraTa) [inline about.html]: - [ ] Please wrap the `navbar` into <nav> tag to keep your HTML semantic 😉 
- [ ] In mobile screens users can still scroll the  body of the document, this is not a good practice, at some point, even some sections come into view as shown below in video. I suggest you to add `overflow: hidden;` to `body` tag when  users click the mobile menu is open:

https://user-images.githubusercontent.com/77038610/152370049-e32528ef-ccfc-4714-abd8-ad0793191eb6.mp4

File: index.html [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+  <div id="mobile-menu">
+    <div id="close-btn">X</div>
+    <a href="./about.html">About</a>
+    <a href="#main-program">Program</a>
+    <a href="#">Join</a>
+    <a href="#">Sponsor</a>
+    <a href="#">News</a>
+    <a href="#">CB Campaing</a>
+  </div>
```
Comment (ShoiraTa) [inline index.html]: - [ ] As I have already mentioned, please wrap this section into `<nav>` tag

File: index.html [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+            <a href="#" id="CB-button">CB Campaing</a>
+          </div>
+        </div>
+      </div>
+    </div>
+  </header>
+
+  <main>
+    <div id="titles">
+      <h2 id="upper-title">"Hello! Creative world"</h2>
```
Comment (ShoiraTa) [inline index.html:59]: - [ ] Please notice that the content is not properly alighned in the center, on the right side there is white blank space of around 1rem, please align sections in the center: 
image

- [ ] Please try to stick to `behance` design while adding margins and padding, for example: 

Behance  | Yours
------------- | -------------
image | image

> As you can see the section with orange background is too close to the border in your document.

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @AlejandroNeira98,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

- [ ] Please make sure to paste the link to video presentation of your project with the maximum length of the video - 5 minutes: 
-  The video should contain a clear and objective description about the project in professional English. The topic and purpose of the project should be made clear.
- The video should contain a demo of the project, showing the pages in different screen sizes.
- The video should contain a highlight about a piece of the code you found interesting or proud of making.

_Check more comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (mateo951) [reply index.html:59]: Addressed 😎

File: index.html [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+
+  <footer>
+    <div class="footer_container">
+      <img src="./images/logo.png" alt="logo" id="footer-logo">
+      <div>
+        <p>2022 Business Commons Colombia. Some Rights Reserved</p>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent cursus risus in ante dapibus faucibus. Sed a elit volutpat massa venenatis facilisis. In aliquet ornare augue ut efficitur.</p>
+      </div>
+    </div>
+  </footer>
```
Comment (mateo951) [inline index.html:126]: Addressed ✔ the footer was included correctly

File: index.html [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+    <h2>Partner</h2>
+    <div class="orange-line"></div>
+    <div id="partners">
+      <img src="./images/partners/mozilla.png" alt="Mozilla">
+      <img src="./images/partners/Google.png" alt="Google">
+      <img src="./images/partners/navar.png" alt="Naver">
+      <img src="./images/partners/daumkakao.svg" alt="Daumkakao">
+      <img src="./images/partners/airbnb.png" alt="airbnb">
+    </div>
+  </div>
```
Comment (mateo951) [inline index.html:116]: Addressed 👌

File: about.html [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+
+  <nav id="mobile-menu">
+    <div id="close-btn">X</div>
+    <a href="./about.html">About</a>
+    <a href="#main-program">Program</a>
+    <a href="#">Join</a>
+    <a href="#">Sponsor</a>
+    <a href="#">News</a>
+    <a href="#">CB Campaing</a>
+  </nav>
```
Comment (mateo951) [inline about.html:20]: Addressed 🙌 Now your semantics are much better 👍

File: mobile-menu.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
+const menubtn = document.getElementById('menu-icon');
+const menu = document.getElementById('mobile-menu');
+const closebtn = document.getElementById('close-btn');
+
+menubtn.addEventListener('click', () => {
+  menu.style.display = 'flex';
+  document.body.style.overflow = 'hidden';
```
Comment (mateo951) [inline mobile-menu.js:7]: Addressed 👌 The issue with the scrolling is no longer happening 🔥

**Review by mateo951 — APPROVED** [review]
### Hi @AlejandroNeira98, 

You have done a fantastic job with your capstone 🔥 Also, you did very well fixing all the changes requested by your previous reviewer 😎
In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it 🔥🔥🔥

### Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**